src/runners/handlers/slack.py

```diff
@@ -67,19 +66,31 @@ def handle(
     slack_token = vault.decrypt_if_encrypted(slack_token_ct)
 
     sc = SlackClient(slack_token)
-
     # otherwise we will retrieve email from assignee and use it to identify Slack user
     # Slack user id will be assigned as a channel
 
     title = alert['TITLE']
 
     if recipient_email is not None:
-        result = sc.api_call("users.lookupByEmail", email=recipient_email)
```

Comment:
it seems result *used* to be the response to `users.lookupByEmail` but now it is the response to `conversations.open` -- could you read the rest of the function to make sure it makes sense, still, and update e.g. the `log.error` on line 96 with the new meaning?